### ISSUE TITLE
v55.0.0: `plugin:jsdoc/recommended` not found

### ISSUE THREAD

**Issue by kyoshino** [gajus/eslint-plugin-jsdoc#1478]
ESLint stopped working after upgrading to `eslint-plugin-jsdoc@55.0.0`. The terminal says the `recommended` plugin is missing:

```
> eslint .

Oops! Something went wrong! :(

ESLint: 8.57.1

ESLint couldn't find the config "plugin:jsdoc/recommended" to extend from. Please check that the name of the config is correct.

The config "plugin:jsdoc/recommended" was referenced from the config file in "/Users/Kohei/Sites/sveltia-cms/.eslintrc.yaml".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.
```

Note: I cannot upgrade to ESLint v9 because `eslint-config-airbnb-base` doesn’t support v9. But I believe `eslint-plugin-jsdoc` itself still supports v8, as per the [README](https://github.com/gajus/eslint-plugin-jsdoc?tab=readme-ov-file#eslintrc) and `peerDependencies`:

https://github.com/gajus/eslint-plugin-jsdoc/blob/0b0a0582a5ad77284c146e01f5347b3fe541e80f/package.json#L124

## Expected behavior

ESLint runs normally.

## Actual behavior

ESLint doesn’t run at all.

## ESLint Config

```yaml
plugins:
  - jsdoc
extends:
  - plugin:jsdoc/recommended
```

Full config: https://github.com/sveltia/sveltia-cms/blob/main/.eslintrc.yaml

## Environment

- Node version: 24.7.0
- ESLint version 8.57.1
- `eslint-plugin-jsdoc` version: 55.0.0

**Comment by kyoshino** [comment]
[v57.0.0](https://github.com/gajus/eslint-plugin-jsdoc/releases/tag/v57.0.0) reintroduced the issue 😿

**Comment by brettz9** [comment]
@kyoshino : Sorry about that. Try with 57.0.7. Appears to be working for me.
